### ISSUE TITLE
replace relative paths to data dir in notebooks

### ISSUE THREAD

**Issue by vigsterkr** [shogun-toolbox/shogun#3697]
currently in all of our ipython notebooks the data files are referenced by a simple `../../../data/<filename>` relative path, like for example this:
```
shogun_feats_linear = RealFeatures(CSVFile('../../../data/toy/classifier_binary_2d_linear_features_train.dat'))
```
this is causing a lot of trouble if you copy the notebook somewhere else.

in this entrance task, one should replace all these relative paths with an environment variable called `SHOGUN_DATA_DIR`, which then would have a default value if it's not set.

the fix should be applied to almost every ipython notebook here: https://github.com/shogun-toolbox/shogun/tree/develop/doc/ipython-notebooks

**Comment by amalad** [comment]
Hi! Can I take up this issue?

**Comment by karlnapf** [comment]
No need to ask :)

**Comment by deveshnag1** [comment]
so the ../../../data should be replaced by an os.getenv call right?

**Comment by deveshnag1** [comment]
@vigsterkr Is this what you had in mind?
https://github.com/deveshnag1/shogun/blob/notebook_relative/doc/ipython-notebooks/classification/Classification.ipynb

**Comment by vigsterkr** [comment]
@deveshnag1 mmm in a way yes but as you can see in the meanwhile this has been addressed by somebody else in the https://github.com/shogun-toolbox/shogun/pull/3753 pr

**Comment by vigsterkr** [comment]
fixed in #3753
